msgcntr/messageforums-app/src/webapp/jsp/discussionForum/area/dfWatchSettings.jsp

```diff
@@ -20,26 +20,42 @@
                   var menuLinkSpan = menuLink.closest('span');
                   menuLinkSpan.addClass('current');
                   menuLinkSpan.html(menuLink.text());
+                  $('#revise\\:emailNotificationOption1, #revise\\:emailNotificationOption2, #revise\\:emailNotificationOption3').each(function() {
+                       let label = $('#revise\\:notify_for_all_postings_head').text() + " " + $(this).next().text();
+                       $(this).attr('aria-label', label);
+                  });
               });
           </script>
 				<%@ include file="/jsp/discussionForum/menu/forumsMenu.jsp" %>
 <!--jsp/discussionForum/area/dfWatchSettings.jsp-->
 				<div class="page-header">
-					<h1><h:outputText value="#{msgs.watch_forums_options}" /></h1>
+					<h1 id="notifications_options_head"><h:outputText value="#{msgs.watch_forums_options}" /></h1>
 				</div>
-				<p class="instruction">
+				<div>
 					<h:outputText escape="false" value="#{msgs.watch_forums_options_instruction}" />
-				</p>
-				<h:panelGroup layout="block" styleClass="indnt1">
-					<h:panelGroup styleClass="checkbox">
-						<h:selectOneRadio layout="pageDirection" id="emailNotificationOption"  value="#{ForumTool.watchSettingsBean.emailNotification.notificationLevel}">
-							<f:selectItem itemValue="2" itemLabel="#{msgs.notify_for_all_postings}"/>
-							<f:selectItem itemValue="1" itemLabel="#{msgs.notify_for_postings_to_my_thread}"/>
-							<f:selectItem itemValue="0" itemLabel="#{msgs.notify_none}"/>
-						</h:selectOneRadio>
-					</h:panelGroup>
-				</h:panelGroup>
-
+					<h:outputText  styleClass="sak-banner-info" escape="false" value="#{msgs.watch_forums_options_instruction_note}" />
+                    <f:verbatim><br/></f:verbatim>
+                </div>
+				<h:panelGroup>
```

Comment:
please make the spacing more consistent

Reply:
Updated the changes with consistent spacing. thanks.